src/templates/base/2019/contributors.html

```diff
@@ -262,27 +271,37 @@ <h1 class="title title-lg">
   </div>
   <div class="contributors {{ ' '.join(config.teams.keys()) }}">
     <div class="filter-container">
-      <p>
+      <p aria-live="polite" aria-atomic="true">
         {{ self.filter_by_team() }}
       </p>
-        {% for id, team in config.teams.items() %}
-          <button class="{{ id }}" type="button">{{ localizedTeamNames[id] if localizedTeamNames[id]|length else team.name }}</button>
+      <ul>
+        {% for id, team in localizedTeamNames.items() | accentless_sort() %}
+        {% if id in config.teams %} <!-- note not all team translations may be in this years config -->
+        <li>
+          <button class="{{ id }}" type="button" aria-pressed="false" title="{{ self.filter_by() }} {{ localizedTeamNames[id] if localizedTeamNames[id]|length else team.name }}">
```

Comment:
Great use of `aria-pressed` 😄 

Out of scope of this PR, but one thing I really like doing is tying visual presentation to semantic state. `[aria-pressed]` makes a great styling hook.

Reply:
Yeah I started refactoring the CSS/JS but was growing in scope so decided to keep it as is for this PR for now so could concentrate on the accessibility problem. Agree we should come back to this later though.